html-block-and-inline/html-block-and-inline-quiz-notes.md

```diff
@@ -0,0 +1,39 @@
+# html-block-and-inline-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- How do block-level elements affect the document flow?
+
+- How do inline elements affect the document flow?
+
+- What are the default width and height of a block-level element?
+
+- What are the default width and height of an inline element?
```

Comment:
Don't forget to answer the quiz questions!

Reply:
my sincerest apologies